core/model/src/main/java/org/eclipse/rdf4j/model/util/Models.java

```diff
@@ -32,6 +24,15 @@
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
 
+import org.eclipse.rdf4j.model.BNode;
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.Literal;
+import org.eclipse.rdf4j.model.Model;
+import org.eclipse.rdf4j.model.Resource;
+import org.eclipse.rdf4j.model.Statement;
+import org.eclipse.rdf4j.model.Value;
+import org.eclipse.rdf4j.model.impl.LinkedHashModel;
```

Comment:
Sorry for the import reorg, I've temporarily been using IntelliJ instead of Eclipse and obviously hit the wrong key combo somewhere.